Backend/agent/handlers/blood_pressure.py

```
@@ -0,0 +1,16 @@
+def handle(query, user_context):
+    db = open_db()
+    try:
+        systolic = 120
+        diastolic = 80
+        
+        db.execute(
+            'INSERT INTO blood_pressure_logs (systolic, diastolic) VALUES (?, ?)',
+            (systolic, diastolic)
+        )
+        db.commit()
+        return f"Recorded BP: {systolic}/{diastolic}."
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)